usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -407,6 +407,8 @@
         "partition_column_type": "numeric",
         "is_partition_column_unique": False,
         "delta_table_create_sql": transaction_download_schema,
+        "delta_table_create_options": {"delta.enableChangeDataFeed": True},
+        "delta_table_create_partitions": ["awarding_agency_code", "is_fpds", "action_date_fiscal_year"],
```

Comment:
One test case if failing because `transaction_download` is the only table spec with these properties, so they're not matching:
- `delta_table_create_options`
- `delta_table_create_partitions`